### ISSUE TITLE
Give access to standard objects

### ISSUE THREAD

**Issue by FabienTaillon** [mshanemc/shane-sfdx-plugins#52]
When no `--object` flag is passed, all objects are supposed to have access rights, but this only works for custom objects.

**Comment by mshanemc** [comment]
So even if the standard objects are in local source, they're being ignored?

**Comment by FabienTaillon** [comment]
Yes. I think this is because the code is only handling objects with a '__c' , '__e' and '__b' suffix, [here](https://github.com/mshanemc/shane-sfdx-plugins/blob/master/src/commands/shane/permset/create.ts#L200).

I tried to add standard objects, but same as what I did for fields, I need to use describe because not all objects accept all type of permissions. For instance, PriceBook doesn't allow View All and Modify All.

Then I need to understand what in the describe can help me deduct these values. As I was running out of time I just reverted back, submitted my PR as is, and created an issue to remember. I may be able to fix it myself later :)
